Vostok.ClusterConfig.Core/Parsers/FileParser.cs

```diff
@@ -27,7 +32,7 @@ public ObjectNode Parse(FileInfo file)
 
                 using (var stream = new FileStream(file.FullName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite | FileShare.Delete))
                 {
-                    return parser.Parse(file.Name.ToLower(), new FileContent(stream, (int)fileSize));
+                    return parser.Parse(file.Name.ToLower(), new FileContent(stream, (int) file.Length));
```

Comment:
Now we have two calls to `FileInfo.Length` per file (one here and one inside `FileSizeLimiter`). Probably not crucial, but still undesirable if there's no internal caching.